packages/react/src/lib/config/supportedWallets.ts

```diff
@@ -0,0 +1,85 @@
+import { SeiWallet } from '../provider';
+
+export const FIN_WALLET: SeiWallet = {
```

Comment:
Would it make sense to move these wallet objects into the `core` package? Don't think this information is react-specific

Reply:
I kinda like them in React because that's the only place we're using them. I feel like core is most general and react is just things for the react library.

Reply:
Ya, I'm just thinking in the future if we want, these objects can be used by themselves without the react components. But this is fine for now, we can move it later if needed